cmd/influxd/config.go

```diff
@@ -446,3 +451,34 @@ func (g *Graphite) LastEnabled() bool {
 
 // maxInt is the largest integer representable by a word (architeture dependent).
 const maxInt = int64(^uint(0) >> 1)
+
+type OpenTSDB struct {
+	Addr string `toml:"address"`
+	Port int    `toml:"port"`
+
+	Enabled         bool   `toml:"enabled"`
+	Database        string `toml:"database"`
+	RetentionPolicy string `toml:"retention-policy"`
+}
```

Comment:
Sorry, not sure what you mean here. Each line already has the toml struct tag, and the entry in the Config struct does too (line 97)